Review Activities 1 and 2

Episki2001 opened the pull request.

Episki2001 committed:
Completed toRna function

Added the accomplished toRna function in the index.js file

Episki2001 committed:
Added test Script for RNA Conversion Challenge

Modified package.json to add test script for the activity

Episki2001 committed:
Accomplished isLeapYear Activity

Modified is:LeapYear function and added test script to package.json

Episki2001 committed:
Added comments to isLeapYear folder

added comments to the index.js file in 00-isLeapYear to describe the function

0917-xyza commented on 00-dnaRnaConversion/index.js:
```diff
@@ -30,4 +30,35 @@ function toRna(input) {
+ * @returns Corresponding RNA Strand
+ */
+function toRna(input) {
+   var result = "";
```
Please refrain from using `var` from now on. Use `let` or `const` from ES6 instead.

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

0917-xyza commented on 00-dnaRnaConversion/index.js:
```diff
@@ -30,4 +30,35 @@ function toRna(input) {
+         case 'A': result += 'U';
+            break;
+         default : return "Error: Invalid Input"
+            // console.log('Error: Invalid Input');
```
always remove unnecessary comments

0917-xyza commented on 00-isLeapYear/index.js:
```diff
@@ -1,8 +1,13 @@
+ */
 function isLeapYear(n) {
-    return n
+    return n % 4 == 0;
```
Please review the definition of a leap year

https://www.kumon.com/resources/leap-year-math/

0917-xyza reviewed (changes requested):
Hi @Episki2001, I have requested changes. Please check per comment for more information.

Also, change name of the pull request (PR) such us 'Solution to XXX challenge'. Make it descriptive but short. Have the PR point to [globe-isg-workplace-dev:Episki2001](https://github.com/globe-isg-workplace-dev/WT-algorithms/tree/Episki2001). Always add also am information in the description of the pull request. Tag me also as one of the reviews. Always 


Thanks

Episki2001 committed:
Merge pull request #1 from globe-isg-workplace-dev/main

add pokemon data

Episki2001 committed:
Update index.js

Episki2001 committed:
Accomplished NumToWord

Completed NumToWord function and added test script to package.json

Episki2001 committed:
Merge branch 'main' of https://github.com/Episki2001/WT-algorithms

Episki2001 committed:
Completed Challenge 4

Added findNames, findNamesByResistance, findByResisrance, avgMaxCP,
and Counted Weaknesses functions as well as the relevant test script in package,json.

Episki2001 committed:
Added comments to challenge 4

Added comments to functions, also commented out the console.log for the findByResistance function

Episki2001 committed:
Resolved Comments from PR Solution 1 and 2

Episki2001 committed:
Resolved issues with Solution 1

Modified isLeapYear function to take into account years that are divisible by 100

Episki2001 committed:
Revert "Resolved issues with Solution 1"

This reverts commit 7ce376e1cd2044d5b2a3b215e7a8f9c46f8044c6.

Episki2001 committed:
Modified var to let in pokedex

Modified instances of var and let in pokedex
